composer_conflict_renovate.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+use Github\AuthMethod;
+use Github\Client;
+use LTS\WordpressSecurityAdvisoriesRenovator\Controllers\GithubApiController;
+use LTS\WordpressSecurityAdvisoriesRenovator\Services\ComposerConflictsRenovator;
+
+require 'vendor/autoload.php';
+
+define('BOT_PERSONAL_ACCESS_TOKEN', getenv('BOT_PERSONAL_ACCESS_TOKEN'));
+define('REPO_OWNER', getenv('REPO_OWNER'));
+define('REPO_NAME', getenv('REPO_NAME'));
+define('API_PAUSE_BETWEEN_ACTIONS_SECONDS', (int)getenv('API_PAUSE_BETWEEN_ACTIONS_SECONDS'));
+
```

Comment:
how about checking/setting php max_execution_time?

```php
ini_get('max_execution_time')
```

Reply:
When starts from a console, it defaults to 0 (no limit). What are your suggestions, 1 hour, 10 minutes, smth like that?